### PR TITLE
Covered more attachments

### PR DESCRIPTION
**Pull request by m7madmagdy**

### CODE REVIEW

**Comment by m7madmagdy** [comment]
Imgur service uploader not tested yet

**Comment by a7madM** [comment]
> Imgur service uploader not tested yet

why ?

File: spec/factories/attachments.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   factory :attachment do
     raw_response { Faker::Lorem.paragraph }
     resource_type { 'image/png' }
-    resource_id { Random.hex(16) }
+    resource_id { Random.hex(2) }
```
[the remark this answers is not in the document]
Comment (m7madmagdy) [reply spec/factories/attachments.rb:7]: الحنكة

**Comment by a7madM** [comment]
> > Imgur service uploader not tested yet
> 
> why ?

**Comment by m7madmagdy** [comment]
> > Imgur service uploader not tested yet
> 
> why ?
بعد التراويح

**Pull request merged**